apps/web/src/hooks/useNewsletterFilters.ts

```
@@ -0,0 +1,50 @@
+    if (timeFilter !== "all") {
+      const now = new Date();
+      const today = new Date(now.getFullYear(), now.getMonth(), now.getDate());
+      
+      filtered = filtered.filter((newsletter) => {
+        const newsletterDate = new Date(newsletter.date);
+        
+        switch (timeFilter) {
+          case "day":
+            return newsletterDate >= today;
+          case "week":
+            const weekAgo = new Date(today);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)